include/PerformanceBenchmarkingUtilities.h

```diff
@@ -11,6 +11,16 @@
 #include <ctime> //TODO:  Move to utiliites
 #include "itkHighPriorityRealTimeProbesCollector.h"
 
+#if ITK_VERSION_MAJOR >= 5
+#include "itkMultiThreaderBase.h"
+using MultiThreaderName = itk::MultiThreaderBase;
+#define PARALLELISM( x ) SetNumberOfWorkUnits( x )
```

Comment:
I was confused about the name. What do you think about SET_PARALLEL_UNITS? or something with SET on it...

Reply:
I agree with Pablo. Readability/understanding are improved with that change. Other than that, I think it's OK.